### ISSUE TITLE
Fereal Druid unlimited free instant healing touch

### ISSUE THREAD

**Issue by WarcraftAcc** [Valhalla-Gaming/Tracker#9192]
**Describe the issue you're having**:
Unlimited free, instant healing touch while passive is active
**Explain how you expect it work**:
Only one free instant healing touch while passive is active

**Comment by MrZlim** [comment]
The spell that causes this is 

http://www.wowhead.com/spell=16974/predatory-swiftness

Predatory swiftness does not disappear after regrowth

Topic should be changed to [Druid][Feral][Spell] Predatory swiftness

**Comment by Fdistvan** [comment]
yes ther is the infinite regrowth   wile  predator is on, u sed the wrong spell  its not healing tuch its regrowht

**Comment by WarcraftAcc** [comment]
Oh crap, edit
